src/ParquetMsg.chpl

```diff
@@ -1925,7 +1926,7 @@ module ParquetMsg {
           st.addEntry(valName, entryVal);
           rnames.pushBack((dsetname, ObjType.PDARRAY, valName));
         } else {
-          var errorMsg = "Null indices only supported on Parquet string columns, not {} columns".format(ty);
+          var errorMsg = "Null indices only supported on Parquet string columns, not {} columns".doFormat(ty);
```

Comment:
so this was in the existing code, but `{}` isn't how format strings work in chapel right? I think someone accidentally did it the python way. I think this needs to be updated to use `%?` instead
```suggestion
          var errorMsg = "Null indices only supported on Parquet string columns, not %? columns".doFormat(ty);
```